assert/assertions_test.go

```diff
@@ -2738,6 +2738,42 @@ func TestErrorAssertionFunc(t *testing.T) {
 	}
 }
 
+func ExamplePanicAssertionFunc() {
+	t := &testing.T{} // provided by test
+
+	tests := []struct {
+		name      string
+		panicFn   func()
```

Comment:
Type should be [`PanicTestFunc`](https://pkg.go.dev/github.com/stretchr/testify/assert#PanicTestFunc).

Reply:
Rebased and updated the type